rpcs3/Emu/Cell/PPUThread.cpp

```diff
@@ -3280,6 +3280,7 @@ bool ppu_initialize(const ppu_module& info, bool check_only)
 	{
 		std::unordered_map<std::string, u64> link_table
 		{
```

Comment:
This doesn't feel like the right way to do this, i think the library function should call the syscall itself

Reply:
> This doesn't feel like the right way to do this, i think the library function should call the syscall itself

Code suggested by @elad335 https://github.com/RPCS3/rpcs3/pull/12597#discussion_r962124997

Reply:
This code is for not breaking the current PPU cache, it's not some wrapper to functions.

Reply:
Aha right, thanks